app/views/projects/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<h1>Projects</h1>
+<div class="table-responsive clearfix">
```

Comment:
i never got your answer on why so many clearfix ?

Reply:
Sir these are for line breaks.

Reply:
Sir Ammad aked me to remove <br> and use clearfix

Reply:
@SumraYasmin `clearfix` here not making sense here. Please go through the below link and use clearfix where needed.
https://getbootstrap.com/docs/4.1/utilities/clearfix/